examples/3.x_api/pytorch/nlp/huggingface_models/language-modeling/quantization/weight_only/run_clm_no_trainer.py

```diff
@@ -44,7 +48,7 @@
                     help="Pad input ids to max length.")
 parser.add_argument("--calib_iters", default=512, type=int,
                     help="calibration iters.")
-parser.add_argument("--tasks", default="lambada_openai,hellaswag,winogrande,piqa,wikitext",
+parser.add_argument("--tasks", default="lambada_openai,hellaswag,winogrande,piqa",
```

Comment:
is there any issue with wikitext?  or just because it reports ppl instead of acc?

Reply:
will update later

Reply:
it's because batch_size=8 will cause oom on wikitext, which is conflict with readme. I will write separate Readme for HPU.